GitHubAssignees.ps1

```diff
@@ -323,12 +323,16 @@ function Remove-GithubAssignee
 
     .EXAMPLE
         Remove-GithubAssignee -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Assignee $assignees
-
```

Comment:
Here and elsewhere: The rest of this module typically keeps a linefeed between the code and the example description.  A couple exceptions managed to creep in over time, but I'd prefer we keep the separation between the code and the explanation.